### PR TITLE
Fix javadoc failure

### PR DESCRIPTION
**Pull request by milaGGL**
Adding / updating comments to pass javadoc lint check

### CODE REVIEW

**Comment by suztomo** [comment]
Assuming you're working on javadoc error fix, you want to focus on the "errors: " lines rather than "warnings: ". Note that the warnings for javadoc may not match Google's coding convention. In that case you don't have to fix warnings.

> Tip: A common mistake is to write simple Javadoc in the form /** @return the customer ID */. This is incorrect, and should be changed to /** Returns the customer ID. */.

https://google.github.io/styleguide/javaguide.html

File: google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public class AggregateQuery {
     this.query = query;
   }
 
-  /** Returns the query whose aggregations will be calculated by this object. */
+  /** @return the query whose aggregations will be calculated by this object. */
```
Comment (tom-andersen) [inline google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java:56]: The sentence starts after the `@return` annotation, so make the first letter a capital. Please do this for other texts as well.

```suggestion
  /** @return The query whose aggregations will be calculated by this object. */
```

**Comment by suztomo** [comment]
You want to focus on the "errors: " lines rather than "warnings: ".

**Comment by milaGGL** [comment]
Re-opening a new PR which only addresses errors, not warnings.

**Comment by milaGGL** [comment]
> You want to focus on the "errors: " lines rather than "warnings: ".

Thank you @suztomo, javadoc test passed after fixing the "errors".

**Pull request closed without merging**